app/models/tag.rb

```diff
@@ -89,6 +91,10 @@ def update_parent(representative)
     update(representative: representative)
   end
 
+  def representative
+    super || self
+  end
+
```

Comment:
True, I've updated most of the methods fetching the reps to check rather for reps_id. This was just a convenient method to always fetch the representative of the tags, which seemed to be useful when doing some larger collections stuff, e.g I think it would be easier to fetch all collections of tags, by calling tags.map(&:representative).